mh/pat.py

```diff
@@ -1781,15 +1815,32 @@ def sendNtcFriendAdd(self, recipient_id, accepted, seq):
         """
         if not accepted:
             return
-        data = b""
-        data += pati.lp2_string(recipient_id)
+
+        # Tell sender their friend request was accepted
+        data_1 = pati.lp2_string(recipient_id)
         friend = pati.FriendData()
-        friend.index = pati.Long(2)
+        friend.index = pati.Long(len(partner_session.get_friends_list()))
+        friend.capcom_id = pati.String(self.session.capcom_id)
+        friend.hunter_name = pati.String(self.session.hunter_name)
+        data_1 += friend.pack()
+        data_1 += struct.pack(">B", accepted)
```

Comment:
There is no need to recreate a variable (`data_1`, `data_2`)  for each packet, just reuse the `data` variable.